lua/buffon/config.lua

```diff
@@ -10,6 +10,10 @@ local default = {
   num_pages = 2,
   open = {
     by_default = true,
+    offset = {
```

Comment:
Remember to update the test. Go to `config_spec.lua` and in the "config" test, add the offset there as well.
This test ensures that the default config is as expected.